tests/runtests.pl

```diff
@@ -5446,8 +5446,9 @@ sub disabledtests {
             if($_ =~ /(\d+)/) {
                 my ($n) = $1;
                 $disabled{$n}=$n; # disable this test number
-                if(! -f "data/test$n") {
-                    print STDERR "WARNING! Non-exiting test $n in DISABLED!\n";
+                my $filepath = getcwd . "/data/test$n";
```

Comment:
I think it should be `$srcdir/data/test$n`

Reply:
My approach was to use an absolute path which turned out to fail...
Will try that one